modules/govuk/manifests/node/s_content_data_api_db_admin.pp

```diff
@@ -31,51 +31,15 @@
   include govuk_env_sync
   include ::govuk::node::s_base
```

Comment:
Did we hit problems having these defined in modules/govuk/manifests/nodes/postgresql_db_admin.pp ?

Reply:
We didn't, but I kept these here based on your [comment](https://github.com/alphagov/govuk-puppet/pull/11345#discussion_r767847067). I see the comment was only on the generic `s_db_admin.pp` but it seems to apply to the other DB admin machines too. I was in two minds about whether or not to keep them in the first place. There's something quite nice about keeping the non-Postgres related stuff out of `postgresql_db_admin.pp`, even if it means a bit more repetition.

Reply:
Yup, that sounds fair. Thanks